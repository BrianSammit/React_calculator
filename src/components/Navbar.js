import { Link } from 'react-router-dom';
import '../App.css';

const Navbar = () => (
  <div className="navbar">
    <h1>Math Magicians</h1>
    <div className="links">
      <Link to="/">Home | </Link>
      <Link to="/quote">Quote of the Day | </Link>
      <Link to="/calculator">Calculator </Link>
    </div>
  </div>
);

export default Navbar;
