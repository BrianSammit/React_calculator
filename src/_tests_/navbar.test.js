import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';
import Navbar from '../components/Navbar';

describe('Home', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <Router>
          <Navbar />
        </Router>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
