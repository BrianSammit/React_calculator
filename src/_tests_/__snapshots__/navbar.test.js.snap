// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders correctly 1`] = `
<div
  className="navbar"
>
  <h1>
    Math Magicians
  </h1>
  <div
    className="links"
  >
    <a
      href="/"
      onClick={[Function]}
    >
      Home | 
    </a>
    <a
      href="/quote"
      onClick={[Function]}
    >
      Quote of the Day | 
    </a>
    <a
      href="/calculator"
      onClick={[Function]}
    >
      Calculator 
    </a>
  </div>
</div>
`;
